WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -80,6 +98,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mLookAndFeelHeader = (LinearLayout) rootView.findViewById(R.id.my_site_look_and_feel_header);
         mThemesContainer = (RelativeLayout) rootView.findViewById(R.id.my_site_themes_container);
 
+        View postsGlow = rootView.findViewById(R.id.postsGlowBackground);
```

Comment:
Why is the animation started here when it's already started in `onActivtyResult`?